src/auth/Login.tsx

```diff
@@ -0,0 +1,79 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import axios from 'axios';
+import useLocalStorage from './localStorage';
+
+interface FormValue {
+  email: string;
+  password: string;
+}
+
+const TOKEN_KEY = 'token';
+
+const Login = () => {
+  const {
+    register,
+    handleSubmit,
+    setError,
+    formState: { isSubmitting, errors },
+  } = useForm<FormValue>();
+  const [storedValue, setValue, removeValue] = useLocalStorage(TOKEN_KEY, '');
+
+  const onSubmitHandler: SubmitHandler<FormValue> = async (data) => {
+    await axios
+      .post('http://kdt.frontend.3rd.programmers.co.kr:5006/login', {
+        email: data.email,
+        password: data.password,
+      })
+      .then((res) => {
+        const token = res.data.token;
+        setValue(token);
+      })
```

Comment:
```suggestion
      .then((res) => setValue(res.data.token))
```
token을 상수로 선언하지 않아도 괜찮을 거 같아요ㅎㅎ